templates/create-accounts.html

```diff
@@ -31,20 +31,20 @@ <h2 class="w-full text-center my-0 py-0">{{_('class')}}</h2>
                     <div class="italic text-xl text-blue-600">{{_('download_login_credentials')}}</div>
                     <div>
                         <div class="flex flex-row items-center gap-4">
-                            <input type="radio" name="download_credentials_checkbox" value="yes" checked>
+                            <input type="radio" id="check_box_yes" name="download_credentials_checkbox" value="yes" checked>
```

Comment:
This seems redundant. Wouldn't it be possible to call the input with name `download_credentials_checkbox` and the corresponding value?

Reply:
I tried getting it by name, but I am not able to get it. Because the name is the same for the checkbox no. And if I change the name the credentials can not be downloaded/